redshift_sqlalchemy/dialect.py

```diff
@@ -12,6 +12,9 @@
 except ImportError:
     pass
 else:
+    from alembic.ddl.base import RenameTable
+    compiles(RenameTable, 'redshift')(postgresql.visit_rename_table)
+
     class RedshiftImpl(postgresql.PostgresqlImpl):
```

Comment:
It's because sqlalchemy uses the value in `__dialect__`, which RedshiftImpl overrides

This is where alembic defines the postgres override: https://github.com/zzzeek/alembic/blob/eb077a6c8d2c468736ba537eb1ab5400c9059aad/alembic/ddl/postgresql.py#L124